koku/api/resource_types/view.py

```diff
@@ -96,6 +96,10 @@ def get(self, request, **kwargs):
                 "path": "/api/cost-management/v1/resource-types/cost-models/",
                 "count": cost_model_count,
             }
+            settings_dict = {
+                "value": "settings",
+                "available": 1,
```

Comment:
Whoops I missed this. I think this needs to be count no?? Because the logic reading this api is probably doing a `.get("count")` so we need this consistent. Additionally should we have the settings path here for consistency too?